src/test/java/blackjack/domain/card/CardTest.java

```diff
@@ -0,0 +1,23 @@
+package blackjack.domain.card;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CardTest {
+    @Test
+    @DisplayName("숫자 계산 테스트")
+    void addNumber() {
+        Card card = new Card(CardPattern.SPADE, CardNumber.SEVEN);
+        assertEquals(card.addPoint(3), 10);
```

Comment:
```suggestion
        assertEquals(10, card.addPoint(3));
```
결과는 동일하지만, 첫 번째 파라미터가 예상, 두 번째 파라미터가 실제 결과입니다 ^_^

Reply:
잘 몰랐네요😅 감사합니다 ㅎㅎㅎ